Description: More -w flags for CFLAGS/CXXFLAGS
Author: Seb James <seb.james@sheffield.ac.uk>
Forwarded: No need to forward.
Last-Update: 2014-02-23

--- a/makefile.brahms
+++ b/makefile.brahms
@@ -284,7 +284,7 @@
 ifeq ($(EASY_BUILD), 1)
 	CFLAGS=-fPIC -ffast-math $(FLAG_PTHREAD) $(FLAG_ARCH) -Wfatal-errors -fno-strict-aliasing
 else
-	CFLAGS=-fPIC -Werror -Wall -ffast-math $(FLAG_PTHREAD) $(FLAG_ARCH) -Wfatal-errors -fno-strict-aliasing -fstack-protector --param=ssp-buffer-size=4
+	CFLAGS=-fPIC -g -Wformat -Werror=format-security -Werror -Wall -ffast-math $(FLAG_PTHREAD) $(FLAG_ARCH) -Wfatal-errors -fno-strict-aliasing -fstack-protector --param=ssp-buffer-size=4
 	LDFLAGS=-Wl,-z,relro
 endif
 
