--- a/makefile.brahms
+++ b/makefile.brahms
@@ -259,6 +259,7 @@
 	# compiler
 	CC=gcc
 	CPPFLAGS=-fvisibility=hidden -fvisibility-inlines-hidden
+	CPPFLAGS=-D_FORTIFY_SOURCE=2
 	CXX=g++ $(CPPFLAGS)
 	NAMEEXE=-o$(SPACE)
 	NAMEOBJ=-o$(SPACE)
@@ -284,7 +285,6 @@
 	CFLAGS=-fPIC -ffast-math $(FLAG_PTHREAD) $(FLAG_ARCH) -Wfatal-errors -fno-strict-aliasing
 else
 	CFLAGS=-fPIC -Werror -Wall -ffast-math $(FLAG_PTHREAD) $(FLAG_ARCH) -Wfatal-errors -fno-strict-aliasing -fstack-protector --param=ssp-buffer-size=4
-	CPPFLAGS=-D_FORTIFY_SOURCE=2
 	LDFLAGS=-Wl,-z,relro
 endif
 
